check_kernel

```diff
@@ -137,7 +150,11 @@ def installed_kernel_versions_fedora():
 
 
 def installed_kernel_version():
-    return sorted(installed_kernel_versions())[-1]
+    installed = sorted(installed_kernel_versions())
+    if not installed:
+        print('KERNEL UNKNOWN - unable to determine running kernel version')
```

Comment:
Dear Mike,

thanks for merging. I just discovered a minor typo here. The message should be "_unable to determine **installed** kernel version_". Can you fix it?

With kind regards,
Andreas.

Reply:
Thanks for spotting! Fixed in https://github.com/neingeist/check_kernel/commit/43ba2f79b102a78d14302478e60ec071f3203f3f

Reply:
That was quick. Thank you!